widget/playground/src/components/MultiList/MultiList.tsx

```diff
@@ -194,6 +198,7 @@ export function MultiList(props: MultiListPropTypes) {
         <StyledButton
           type="primary"
           size="medium"
+          disabled={!selectedItems.length}
```

Comment:
I think disable logic should be based on all selected tokens length not only current selected blockchains

Reply:
fixed.